<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_interface_desktop</class>
 <widget class="QWidget" name="page_interface_desktop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Embedded Utilities</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list_desktop_plugins">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QToolButton" name="tool_desktop_addplugin">
       <property name="text">
        <string notr="true">add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tool_desktop_rmplugin">
       <property name="text">
        <string notr="true">rem</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="check_desktop_autolaunchers">
       <property name="text">
        <string>Display Desktop Folder Contents</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_media_icons">
       <property name="text">
        <string>Display Removable Media Icons</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
